.travis.yml

```diff
@@ -3,5 +3,7 @@ scala:
   - 2.11.6
 services:
   - mongodb
+jdk:
+  - oraclejdk8
```

Comment:
It does in its current form. We *could* make it work with 7 but that adds some complications.